### PR TITLE
Fast Template Refs: use bind api and enable full screen (#2577)

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
Addresses #2577

- [x] use `pn.bind`
- [x] make *full screen* work
- [x] use accent color for plots.

### CODE REVIEW

File: examples/reference/templates/FastGridTemplate.ipynb [hunk 1]
```diff
@@ -150,6 +154,9 @@
   }
  ],
  "metadata": {
+  "interpreter": {
+   "hash": "7fa204e84b1ce6697eb3048458244c17ccc160da5eb825bd435eb4b5077a9bf7"
+  },
```
Comment (philippjfr) [inline examples/reference/templates/FastGridTemplate.ipynb:159]: Any idea where this came from?

Comment (MarcSkovMadsen) [reply examples/reference/templates/FastGridTemplate.ipynb:159]: Nope

Comment (MarcSkovMadsen) [reply examples/reference/templates/FastGridTemplate.ipynb:159]: Do you agree that the `pn.bind` api probably is the "best api" to communicate to users?

Comment (MarcSkovMadsen) [reply examples/reference/templates/FastGridTemplate.ipynb:159]: Trying out new slogans for the template

image

:-)

Comment (philippjfr) [reply examples/reference/templates/FastGridTemplate.ipynb:159]: Yes, @jbednar and I agree that `pn.bind` is most powerful.

**Comment by philippjfr** [comment]
Got a conflict now and if you could remove that weird hash in the notebook I'll merge.

**Comment by MarcSkovMadsen** [comment]
I've removed the hash @philippjfr

File: examples/reference/templates/FastGridTemplate.ipynb (hunk 1 above)
Comment (MarcSkovMadsen) [reply examples/reference/templates/FastGridTemplate.ipynb]: I've removed the hash.

**Pull request merged**